### PR TITLE
Inspiration page

### PR DESCRIPTION
**Pull request by mirrou**
Starter Inspiration Page UI with an input field, submit, and card to display image pulled from url

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -58,11 +59,16 @@ export const AppHeader = () => (
     <div>
         <a href='./'><img src={logo} alt="logo"className='logo'/></a>
     </div>
+    <div>
+    <div className='avatar-container'>
+      <Avatar shape="square" size="large" src="https://robohash.org/User" />
```
Comment (reggieroby) [inline src/App.js]: the hash should be dynamic based on the current user signed in. im sure that can be done later when the server side is fully implemented and you have a user while navigating.

File: src/components/Pages/Inspiration.js [hunk 2]
```diff
@@ -1,5 +1,63 @@
...
-  return <div>INSPIRATION PAGE</div>;
+const { Meta } = Card;
+
+class PinInput extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { url: '' };
+
+    this.handleChange = this.handleChange.bind(this);
```
Comment (reggieroby) [inline src/components/Pages/Inspiration.js]: very good approach. it took me a long time to figure out this pattern!!!

File: src/components/Pages/Join.js [hunk 3]
```diff
@@ -144,6 +157,10 @@ class RegistrationForm extends React.Component {
           <Button type="primary" htmlType="submit">Join</Button>
         </FormItem>
       </Form>
+
+      <img src={`//robohash.org/${this.props.form.getFieldValue('email')}?size=200x200`} alt="" />
```
Comment (reggieroby) [inline src/components/Pages/Join.js]: this seems to be dynamic while the user types in their email. is that the actual result? not a pressing issue at the present time either way.

File: src/App.js (hunk 1 above)
Comment (mirrou) [reply src/App.js]: Yes! It's just a placeholder as we continue to build.

File: src/components/Pages/Join.js (hunk 3 above)
Comment (mirrou) [reply src/components/Pages/Join.js]: it is dynamic...I'm working on changing that :)

**Pull request merged**